src/vm/arm/exceparm.cpp

```diff
@@ -54,6 +54,35 @@ FaultingExceptionFrame *GetFrameFromRedirectedStubStackFrame (T_DISPATCHER_CONTE
     return (FaultingExceptionFrame*)((TADDR)pDispatcherContext->ContextRecord->R4);
 }
 
+//Return TRUE if pContext->Pc is in DispatchStub
+BOOL IsDispatchStub(CONTEXT *pContext)
```

Comment:
Is there any reason for not handling the resolve stubs as well, like the AdjustContextForVirtualStub does?